app/index.js

```diff
@@ -1,10 +1,5 @@
-/* eslint-disable import/no-extraneous-dependencies */
-/**
- * @format
- */
 import 'fast-text-encoding' // polyfill for TextEncoder and TextDecoder
 import 'react-native-gesture-handler'
-// remove these when updated to react-native 0.65.0
```

Comment:
I tested removing this polyfills, the app broke. So removed the comment that says we should remove them.